scripts/_build/ec.sh

```diff
@@ -8,7 +8,14 @@ then
     exit 1
 fi
 
+EC_ARGS=()
+while read line; do
+    if [[ "$line" != "#"* ]]; then
+        EC_ARGS+=("$line")
+    fi
+done < "$1"
+
 source "$1"
```

Comment:
still want to source the file?

Reply:
`BOARD` still needs to be a env var for the `cp` at the end.